src/host/lfs_file_handler.h

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "ccf/indexing/lfs_types.h"
+#include "ds/messaging.h"
+
+#include <filesystem>
+
+namespace asynchost
+{
+  struct LFSFileHandler
+  {
+    const std::filesystem::path root_dir = ".ccf_lfs";
```

Comment:
We should probably consider naming the index files .index for clarity. Other uses of LFS may use different suffixes to indicate what they contain, but naming them all after the mechanism doesn't seem ideal.

Reply:
Sure, different suffixes/dirs are sensible. I think we want them under a single dir so they can be easily cleaned (if the operator finds some problem with them, they're taking up too much space/wants to symlink them). But I'll call that `.index/` for now.

Reply:
(And of course this dir is something that should eventually be configurable! But I don't think there's any need for that yet)